xfce/libxfce4ui/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem 1.0
 
 name            libxfce4ui
-version         4.16.1
+version         4.18.6
 revision        0
-checksums       sha256  d96946ae5af6bf078dda415419e0021909f763ee0020b42f3e26f603e51585f6 \
-                rmd160  76e6e681c70ab8cfcf0c300487535f50dafc73b4 \
-                size    785811
+checksums       sha256  77dd99206cc8c6c7f69c269c83c7ee6a037bca9d4a89b1a6d9765e5a09ce30cd \
+                rmd160  2c39d90ea51931590b74e361cc923b9b8e94e6be \
+                size    906477
```

Comment:
It also needs this:
```
configure.cflags-append -std=c99
```
Otherwise it will be broken with older compilers.

Reply:
Done